---
permalink: /
title: ""
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

I am an associate professor at the <a href="https://business.ku.edu/">University of Kansas School of Business</a>. Statistics is my research area and I am interested in developing flexible and robust models for high-dimensional data. I received a PhD from the School of Statistics in 2014 at the University of Minnesota. In my dissertation, <a href="https://conservancy.umn.edu/handle/11299/163910">quantile regression model selection</a>, I examined model selection and estimation for quantile regression. I examined
* Partial Linear Models
* Models with missing covariates
* High-dimensional data, including p >> n case (more predictors than samples).  

Before starting my job at KU I was a postdoctoral fellow at the Biostatistics Department at Johns Hopkins working with <a href="https://jilab.org/">Hongkai Ji</a>. My work in Hongkai's lab focused on method development for large data sets and collaboration on genomics problems. As a member of Hongkai's lab I provided data analysis for researchers using genomics data and worked on developing prediction methods for genomics data. 

My collaborative work was with <a href="https://publichealth.jhu.edu/faculty/2518/xiaobin-wang">Xiaobin Wang's group</a>. We studied how genetic data and epigenetic data, such as SNP and  DNA methylation data, associate with different phenotypes. We worked on examining relationships between food allergy for children and their DNA methylation data. We also examined if there is a relationship between preterm birth and the genetic and epigenetic data of the mother. 

My method development work in Hongkai's lab focused on building models to predict DNase-Seq data using gene expression data. This is motivated by the large amounts of public gene expression data that could be potentially used to predict other genomic data. The unique challenges of this problem is the number of response, on the order of 1 million, and number of predictors, around 18,000, are large. However the sample sizes used to build these model remain small, around 50-100. Our methods are focused on creating accurate models that account for the noisy data and are computationally reasonable.     

My training at Minnesota focused on developing methods and proving their estimation properties. My work as a postdoc has focused on collaboration and method development for genetic data. Method development work for my postdoc focused on empirical and scientific validation as opposed to theoretical properties. As a professor at KU I will continue to do collaborative work and develop methods that can be empirically or theoretically justified. In addition I provide software for the methods I have developed. 
